webapp/src/app.tsx

```diff
@@ -822,7 +822,7 @@ export class ProjectView
         let changeHandler = () => {
             if (this.editorFile) {
                 if (this.editorFile.inSyncWithEditor)
-                    pxt.tickActivity("edit", "edit." + this.editor.getId().replace(/Editor$/, ''))
+                    pxt.tickEvent("activity.edit", { editor: this.editor.getId().replace(/Editor$/, '')}, { interactiveConsent: true } );
```

Comment:
This one gets hit in the arcade share page after pressing the `show code` (doesn't get hit in micro:bit immediately); that seems valid as it comes after the user has clicked the button to interact, but just want to double check

Reply:
Good point. Seems ok to me.